### PR TITLE
Implement Read the docs.

### LINKED ISSUE #169: Docs not up to date
The latest version of `cf_units` is currently 2.1.5.  The published docs appear to be only showing v2.0:
https://scitools.org.uk/cf-units/docs/latest/

### PR DESCRIPTION
**Pull request by trexfeathers**
### Description

Adds the necessary config to allow Read the docs to work. I will now set up Read the docs to build from the `main` branch.

### CODE REVIEW

File: requirements/py37.yml [hunk 1]
```diff
@@ -25,3 +25,7 @@ dependencies:
 # dev dependencies
   - pip
   - pre-commit
+
+# docs dependencies
+  - setuptools_scm
```
Comment (bjlittle) [inline requirements/py37.yml:30]: This is more of a setup dependency even though it's used during docs building... just seems odd seeing it categorised here.

**Review by bjlittle — APPROVED** [review]

**Comment by bjlittle** [comment]
Let's see how it goes... we can always tweak any changes as necessary.

But LGTM :+1:

File: requirements/py37.yml (hunk 1 above)
Comment (trexfeathers) [reply requirements/py37.yml:30]: I agree. However without it the build was unable to pick up `cf_units.__version__`.

But when not using Read the Docs, everything works fine with it only listed in `pyproject.toml`!

**Comment by trexfeathers** [comment]
Thanks @bjlittle! It seems to have built fine.

**Pull request merged**